src/common/cssVariables.js

```diff
@@ -0,0 +1,34 @@
+export const blue_light = 'rgba(173, 216, 230, 1)';
+export const grey_dark = 'rgba(128, 128, 128, 1)';
+export const blue_medium = 'rgba(51, 102, 255, 0.5)';
+export const red_dark = 'rgba(255, 0, 0, 1)';
+export const blue_dark = 'rgba(0, 0, 255, 1)';
+export const black_light = 'rgba(51, 51, 51, 1)';
+export const grey_light = 'rgba(107, 128, 150, 1)';
+export const white = 'rgba(	250, 250, 250, 1)';
+export const temperatureTextColor = red_dark;
```

Comment:
Are all of your tempterature texts gonna be red?

Reply:
not just red! RED DARK

[![The Emperial March]](https://www.youtube.com/watch?v=-bzWSJG93P8)